core/View/SecurityPolicy.php

```diff
@@ -16,6 +16,12 @@
  */
 class SecurityPolicy
 {
+    /*
+     * Commonly used rules
+     */
+    const RULE_DEFAULT = "'self' 'unsafe-inline' 'unsafe-eval' data:";
+    const RULE_EMBEDDED_FRAME = "'self' 'unsafe-inline' 'unsafe-eval' data: https:";
```

Comment:
Does the `https` mean we allow any https URL? I suppose heatmaps/session recording could also be used with `http`? would we ned to check something like `if (ProxyHttp::isHttps()) then allow also http`? Not sure if that makes sense.

BTW did you check what happens when a recorded page includes a youtube iframe for example? Would that still work to show the youtube iframe?

Reply:
Perhaps could set both http and https to basically allow anything for the iframe?

I have added a frame-src rule to allow iframes which allows youtube embeds. At this point it's fairly unrestricted in that page except no scripts.